credoai/utils/credo_api_utils.py

```diff
@@ -76,18 +76,30 @@ def get_associated_models(use_case_id):
     return deserialize(submit_request('get', end_point).json())['models']
 
 
-def get_assessment_plan(use_case_id, model_id):
+def get_assessment_spec(assessment_spec_url):
+    """Get the assessment spec
+    
+    The assessment spec includes all information needed to assess a model and integrate
+    with the Credo AI Governance Platform. This includes the necessary IDs, as well as 
+    the assessment plan
+    """
     try:
-        end_point = get_end_point(
-            f"use_cases/{use_case_id}/models/{model_id}/assessment_plans/latest")
-        assessment_plan_id = deserialize(
-            submit_request('get', end_point).json())['id']
+        end_point = get_end_point(assessment_spec_url)
+        downloaded_spec = deserialize(submit_request('get', end_point).json())
+        assessment_spec = {k: v for k,
+                           v in downloaded_spec.items() if '_id' in k}
+    except requests.exceptions.HTTPError:
+        raise IntegrationError("Failed to retrieve assessment spec. Check that the url is correct")
+    try:
+        assessment_plan_id = downloaded_spec['id']
```

Comment:
@kunna , right now I am using the ID of the spec. If you plan to include an "assessment_plan_id" in the payload, as we discussed, let me know and I'll update this.

Reply:
I updated and push the changes.
* there will be `assessment_plan_id` attribute
* request will respond with validate data even if assessment_plan is not defined